go.mod

```
@@ -6,15 +6,15 @@ require (
 	github.com/google/go-cmp v0.6.0
 	github.com/iancoleman/orderedmap v0.0.0-20190318233801-ac98e3ecb4b0
 	github.com/sirupsen/logrus v1.4.2
-	github.com/stretchr/testify v1.4.0 // indirect
-	github.com/xeipuuv/gojsonpointer v0.0.0-20190905194746-02993c407bfb // indirect
-	github.com/xeipuuv/gojsonreference v0.0.0-20180127040603-bd5ef7bd5415 // indirect
 	github.com/xeipuuv/gojsonschema v1.1.0
 )
 
-replace (
-	github.com/alecthomas/jsonschema => github.com/alecthomas/jsonschema v0.0.0-20200127222324-dd4542c1f589
-	github.com/iancoleman/orderedmap => github.com/iancoleman/orderedmap v0.0.0-20190318233801-ac98e3ecb4b0
+require (
+	github.com/konsorten/go-windows-terminal-sequences v1.0.1 // indirect
+	github.com/stretchr/testify v1.4.0 // indirect
+	github.com/xeipuuv/gojsonpointer v0.0.0-20190905194746-02993c407bfb // indirect
+	github.com/xeipuuv/gojsonreference v0.0.0-20180127040603-bd5ef7bd5415 // indirect
+	golang.org/x/sys v0.0.0-20190422165155-953cdadca894 // indirect
 )
 
-go 1.13
+go 1.22
```

Without a specific reason, we should update to latest.
```suggestion
go 1.23
```